iib/workers/tasks/build.py

```diff
@@ -535,9 +536,14 @@ def _prepare_request_for_build(
         with set_registry_token(overwrite_from_index_token, from_index):
             from_index_resolved = _get_resolved_image(from_index)
             from_index_arches = _get_image_arches(from_index_resolved)
+            ocp_version = get_image_label(from_index_resolved, "com.redhat.index.delivery.version")
```

Comment:
Could you please use single quotes throughout? Sorry but we enforce that throughout our code base.